src/special/webpack.js

```diff
@@ -66,18 +84,45 @@ function parseWebpack2(module, deps) {
 
   const mappedLoaders = module.rules.filter((rule) => rule.loaders);
   const mappedUses = mapRuleUse(module);
-  const loaders = getLoaders(deps, lodash.flatten([...mappedLoaders, ...mappedUses]));
-  return loaders;
+  const mapped = lodash.flatten([...mappedLoaders, ...mappedUses])
+  const loaders = getLoaders(deps, mapped);
+  const presets = getBabelPresets(deps, mapped);
+  return [...loaders, ...presets];
+}
+
+function extractEntries(entries) {
+  if (typeof entries === 'string') {
+    return [entries];
+  }
+
+  if (Array.isArray(entries)) {
+    return entries.filter((entry) => typeof entry === 'string');
+  }
+
+  return Object.values(entries)
+    .filter((entry) => entry)
+    .map(extractEntries);
+}
+
+function parseEntries(entries, deps) {
+  return lodash(extractEntries(entries))
+    .flatten()
+    .intersection(deps)
+    .uniq()
+    .value();
 }
 
 export default function parseWebpack(content, filepath, deps) {
   const filename = path.basename(filepath);
   if (webpackConfigRegex.test(filename)) {
-    const module = require(filepath).module || {}; // eslint-disable-line global-require
+    const wpConfig = require(filepath); // eslint-disable-line global-require
```

Comment:
You changed the regex to include .ts config files, wouldn't this fail for a ts file?

Reply:
I am working with TypeScript, and it's working like a charm… :smile:

Reply:
Actually, I've got a doubt. It's working with my project but I'm calling depcheck from a TypeScript file using ts-node. Maybe it'd be a requirement in order to use this feature.

Reply:
Ok so yeah it works for you because of ts-node. 

I think we should revert the ts loading until we find a good way of dealing with this

Reply:
I'll do some more tests directly using the command line.
But anyway, will it hurt to keep this commit? I mean, at worst, if people is not using it with the same requisites as I do, it will fail while reading the config file, and therefore, work as before. So it should be improved, but already can be a useful feature…

Reply:
The cli could fail for users where it didn't fail before. At least could you make a PR and make it so that it uses the `tryRequire`?

Reply:
So, after my tests, I can confirm that:

- when using `ts-node`, it works well, even with command line (I tested with `npx ts-node ~/depcheck/bin/depcheck.js --specials=bin,tslint,webpack`);
- when executing depcheck with `node`, the webpack configuration file is ignored, but the rest of the process is working correctly.

Reply:
Great then, nothing broke :)